modules/kit-elements-defaults/module.php

```diff
@@ -34,7 +34,10 @@ private function enqueue_scripts() {
 		wp_enqueue_script(
 			'elementor-kit-elements-defaults-editor',
 			$this->get_js_assets_url( 'kit-elements-defaults-editor' ),
-			[],
+			[
+				'elementor-common',
+				'elementor-editor-modules',
```

Comment:
```suggestion
				'elementor-editor-modules',
				'elementor-editor-document',
```

Seems like this one helps, not sure what is the diffrence